app/design/frontend/Magento/blank/requirejs-config.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
I'd say that adding `requirejs-config.js` at theme root should be considered as anti-pattern and Magento core shouldn't promote it, b/c it's against "everything is a module" aproach.

Also, by default this config file is stored in `Theme` module of `Magento` vendor, so following files structure, it should be added as `app/design/frontend/Magento/blank/Magento_Theme/requirejs-config.js` and files JS files should be moved from `app/design/frontend/Magento/blank/web/js` to `app/design/frontend/Magento/blank/Magento_Theme/web/js`.